liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -135,6 +135,7 @@ public class Main {
     protected String rollbackScript;
     protected Boolean rollbackOnError = false;
     protected List<CatalogAndSchema> schemaList = new ArrayList<>();
+    protected String format;
 
     private static int[] suspiciousCodePoints = {160, 225, 226, 227, 228, 229, 230, 198, 200, 201, 202, 203,
```

Comment:
This isn't a requested change or even related directly to this PR, but what are these suspicious code points? I've seen them in multiple code reviews but have never asked. What makes them suspicious?

Reply:
It was introduced as part of DAT-3464. It is used to identify non-standard characters and advise the user that they may cause issues and/or brake liquibase execution.